src/test/java/teammates/test/automated/FeedbackSessionsEmailTaskQueueCallback.java

```diff
@@ -0,0 +1,31 @@
+package teammates.test.automated;
+
+import static org.testng.AssertJUnit.assertNotNull;
+import static org.testng.AssertJUnit.assertTrue;
+
+import java.util.HashMap;
+
+import teammates.common.util.Const;
+import teammates.common.util.HttpRequestHelper;
+import teammates.common.util.Const.ParamsNames;
+
+import com.google.appengine.api.urlfetch.URLFetchServicePb.URLFetchRequest;
+
+@SuppressWarnings("serial")
+public class FeedbackSessionsEmailTaskQueueCallback extends BaseTaskQueueCallback {
```

Comment:
Either include or eliminate the `TaskQueue` part in all of these classes?
I see you eliminated it from `SubmissionsAdjustmentTaskQueueCallback`, but it's still present here.